README.md

```diff
@@ -0,0 +1,83 @@
+# PyVista Tutorial
+
+## Abstract
+
+This tutorial will introduce the 3D visualization using PyVista (https://github.com/pyvista/pyvista).
+PyVista is 3D plotting and mesh analysis through a streamlined interface for the Visualization Toolkit (VTK).
+PyVista uses VTK (http://www.vtk.org) under the covers and provides a very Python-friendly API.
+
+At the end of the tutorial users will be able to
+
+- Use PyVista effectively to plotting 3D visualizations
+- Understand the basic class structure of PyVista.
+
+## Keywords
+
+[visualization](https://github.com/topics/visualization)
+[meshviewer](https://github.com/topics/meshviewer)
+[vtk](https://github.com/topics/vtk)
+[3d](https://github.com/topics/3d)
+
+## Other Information and Files
+
+### Tutorial Description.
+Same with Abstract.
```

Comment:
1. If it's the same as the abstract we should probably copy the abstract verbatim here, but we can do that when everything else is done so that you don't have to duplicate changes all the time.
2. The abstract in its current state doesn't seem to include "the target audience, the expected level of knowledge prior to the class", so either the abstract should be amended, or adding this information to the "Tutorial Description".

Reply:
Thanks. Now we added content.
https://github.com/pyvista/pyvista-tutorial/blob/doc/add-readme/README.md#tutorial-description